dialog/file.go

```diff
@@ -506,6 +506,14 @@ func (f *FileDialog) SetFilter(filter storage.FileFilter) {
 	}
 }
 
+// PresetFileName sets the filename in a FileSaveDialog.
+// This is normally called before the dialog is shown.
+func (f *FileDialog) PresetFileName(fileName string) {
```

Comment:
`Preset` here does not seem required - it is not happening before anything in particular - why did you decide not to use `SetFileName`?

Reply:
Because this will usually be called before Show() so it "presets" the filename before revealing the dialog. However SetFileName seems fine as well

Reply:
Want me to rename to `SetFileName`?

Reply:
Yes please - "Preset" asserts usage expectations that are not met in what the code does. Plus we don't use that word elsewhere.

P.S. Can you please add tests? I would think 1 showing that it works called before show, and another that shows after?